src/chain.cpp

```diff
@@ -7,6 +7,13 @@
 
 using namespace std;
 
+uint256 CBlockIndex::GetSaplingAnchorEnd() const {
```

Comment:
We shouldn't need to do anything fancy when fetching this, so we shouldn't need a getter. Just access `pindex->hashFinalSaplingRoot`.